endpoints.go

```diff
@@ -132,6 +135,35 @@ var (
 		return EndpointMessageReactions(cID, mID, eID) + "/" + uID
 	}
 
+	EndpointApplicationGlobalCommands = func(aID string) string {
+		return EndpointApplication(aID) + "/commands"
+	}
+	EndpointApplicationGlobalCommand = func(aID, cID string) string {
+		return EndpointApplicationGlobalCommands(aID) + "/" + cID
+	}
+
+	EndpointApplicationGuildCommands = func(aID, gID string) string {
+		return EndpointApplication(aID) + "guilds/" + gID + "/commands"
```

Comment:
It looks like you dropped a slash before `guilds`.

Reply:
That might be the problem I was experiencing.

Reply:
Nevermind, the issue I was having was separate. However, I do remember something about that method causing a 404 so adding the beginning slash will probably fix that specific issue.

Reply:
Fixed that, thank you for finding this little, but yet very important bug.